front/src/components/ResourceDetail.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Link,
+  Box,
+} from '@mui/material';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import type { Resource } from '@shared_types';
+
+interface ResourceDetailProps {
+  resource: Resource | undefined;
+}
+
+const ResourceDetail: React.FC<ResourceDetailProps> = ({ resource }) => {
+  if (!resource) {
+    return <Typography>You must choose a resource</Typography>;
+  }
+  const imageUrl = resource.image
+    ? `data:image/jpeg;base64,${resource.image}`
+    : '';
+
+  return (
+    <Box
+      sx={{
+        position: 'absolute',
+        top: '50%',
+        left: '50%',
+        transform: 'translate(-50%, -50%)',
+        width: '80%', // Adjust the width for responsiveness
+        maxWidth: 600, // Limit max width for better aesthetics
+        backgroundColor: 'background.paper',
+        boxShadow: 24,
+        borderRadius: 2,
+        p: 4,
+      }}
+    >
+      <Card
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          gap: 2,
+        }}
+      >
+        {/* Displaying Larger Image */}
+        <CardMedia
+          component='img'
+          image={
+            typeof imageUrl === 'string'
+              ? imageUrl
+              : `data:image/jpeg;base64,${imageUrl}`
+          }
+          alt={resource.category}
+          sx={{
+            borderRadius: 1,
+            maxHeight: 300,
+            objectFit: 'cover',
+          }}
+        />
+
+        {/* Content */}
+        <CardContent>
+          <Typography
+            variant='h5'
+            textAlign='center'
+            gutterBottom
+            sx={{ fontWeight: 'bold' }}
+          >
+            {resource.category}
+          </Typography>
+          <Typography variant='body1' paragraph>
```

Comment:
Prefer `component` property over deprecated `paragraph` (https://mui.com/material-ui/migration/migrating-from-deprecated-apis/#paragraph):

```suggestion
          <Typography variant='body1' component='p'>
```